docs/pages/database-access/guides/ha.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Database Access High Availability
+description: How to configure Teleport Database Access in HA configuration
+---
+
+# Database Access High Availability
+
+You can deploy Database Access in a highly available configuration in a couple
+of common ways: separate replicas and combined replicas. Both of those revolve
+around pointing multiple Database Services to the same database instance.
```

Comment:
One general request I have (that is in no way related to the other great changes here) involves using some of the new includes: `docs/pages/includes` (and factoring out reused configuration examples, warnings, notes, and steps into new ones as well).

We've been trying to add backup warnings here, for example. It's an obvious best practice but a good idea to include (almost all docs do this). We will also eventually upgrade all of our includes using some new UI/UX components:

1. https://github.com/gravitational/next/issues/275
2. https://github.com/gravitational/next/issues/298

This will not only make our message more consistent, it makes reviewing changes a little less stressful since it reduces the amount of text, also makes it easier to uniformly update the same items.

Reply:
Good call, I was thinking about making some partials as well. I'll take a look if anything can be factored out. The issue is that the commands differ a little bit between the guides - do you know if these includes can be parameterized?

Reply:
I've moved some duplicated sections into includes. Others can't be factored out easily without some parameterization support.